dagshub/data_engine/model/query_result.py

```diff
@@ -401,6 +404,74 @@ def download_binary_columns(
             num_proc=num_proc,
         )
 
+    def get_predictions(
+        self,
+        repo,
+        name,
+        version="latest",
+        pre_hook=lambda x: x,
+        post_hook=lambda x: x,
+        batch_size=1,
+        metadata_column="annotation",
+        return_predictions=True,
+    ):
+        """
+        Sends all the datapoints returned in this QueryResult as prediction targets for an MLFlow model registered on DagsHub.
+
+        Args:
+            open_project: Automatically open the Label Studio project in the browser
+            ignore_warning: Suppress the prompt-warning if you try to annotate too many datapoints at once.
+            fields_to_embed: list of meta-data columns that will show up in Label Studio UI.
+             if not specified all will be displayed.
+            fields_to_exclude: list of meta-data columns that will not show up in Label Studio UI
+            repo: repository to extract the model from
+            name: name of the model in the mlflow registry
+            version: (optional, default: 'latest') version of the model in the mlflow registry
+            pre_hook: (optional, default: identity function) function that runs before datapoint is sent to the model
+            post_hook: (optional, default: identity function) function that converts mlflow model output
+            to the desired format
+            batch_size: (optional, default: 1) function that sets batch_size
+            metadata_column: (optional, default: 'prediction') write prediction results to metadatalogged in data engine.
```

Comment:
1) We're not using `column`, we're trying to use `field` when talking about DE to prevent confusion with pandas dataframes.
2) Until reading the code this particular argument didn't make much sense to me. I think it'd be better as something like `log_to_field` or smth like that. Basically need to specify that this is actually an output and not an input.